### DISCUSSION TITLE
After installing, what next?

### DISCUSSION THREAD

**Discussion by nowodev** [coollabsio/coolify#107] [Q&A] [answered]
Firstly, thanks for this wonderful product.

I've successfully installed coolify but I'm confused on the type of domain I'm supposed to put.

Do I buy a new domain and put the url or what exactly?

**Comment by andrasbacsai** [comment, marked as answer]
Hey! I think you installed the first version of coolify. If I'm right, please install v2 first.

I do not fully understand your question. What you would like to do with Coolify?

**Reply by nowodev** [reply]
Hey man, so I was just trying to play with it to see how it works.
But I got confused about the domain part.
What domain exactly am I supposed to use?
And I'm trying to test a Laravel Deployment

**Reply by narduin** [reply]
You can use any domain you own that has its dns pointing to your coolify server's ip.

I was able to use a subdomain of my main domain like `https://deploy.domain.tld` for the coolify install.
Coolify automatically generated a TLS cert. Very smooth and fully automated.

I was then able to use any domain/subdomain (related or not to the main coolify domain) to deploy applications or services (assuming dns settings are correct).

So my working settings are:
coolify domain -> `https://deploy.domain.tld`
app domain -> `https://other-domain.tld`
service domain -> `https://yet.another-domain.tld`

**Reply by nowodev** [reply]
Great. I'll give it a try. 
Thanks
